project-base/src/Resources/scripts/frontend/cart/cartRecalculator.js

```diff
@@ -63,6 +63,7 @@
 
                 Shopsys.register.registerNewContent($mainContent);
                 Shopsys.register.registerNewContent($cartBox);
+                Shopsys.lazyLoadCall.inContainer('.js-cart');
```

Comment:
I think that we can start thinging about use Register.callback in lazyLoadCall component. What do you think about it?

Reply:
I like the idea @pesektomas, nice point :+1:

Reply:
Discussed with @pesektomas and refactored. All these call were removed, and added register.callback to lazyload function